config/v1/custom.network.testsuite.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "[CustomNoUpgrade] Network"
+crd: 0000_10_config-operator_01_network-CustomNoUpgrade.crd.yaml
+tests:
+  onCreate:
+    - name: Should be able to set status conditions
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Network
+        spec: {} # No spec is required for a Network
+        status:
```

Comment:
No status subresource? Odd

Reply:
`network.config.openshift.io` does not have the status subresource.
We considered adding to it here but decided not to as it could affect the existing client that updates the status without using the status subresource (it would not work).

I don't think this is tightly coupled with this change and we can think of addressing it separately if needed.

Reply:
Yeah sure, don't add it here and we can't change it right now, so will have to deal with it another time